component.html

```diff
@@ -134,6 +135,146 @@ <h3 class="avatar-heading">Alert</h3>
+  style="width: 100%; height: 398px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+    </section>
+
+
+    <!-- --- Button ---- -->
+
+    <section class="main-title" id="buttons">
+      <h2 class="avatar-heading">Button</h2>
+      <p class="component-navbar-item">Buttons are also called as call to action. We have range of buttons and their states. You may use a tag or button element, you need to add respective classes, and you are good to go.</p>
+      <h3 class="avatar-heading">Primary Button</h3>
+      <p class="component-navbar-item"> 
+        Whenever you want your user to click on a link or button, use primary style buttons.  
+      </p>
+
+      <div class="button-items">
+        <p>
+          <button class="btn btn-primary btn-round-1">Primary</button>
+          <button class="btn btn-success btn-round-1">Success</button>
+          <button class="btn btn-warning btn-round-1">Warning</button>
+          <button class="btn btn-danger btn-round-1">Danger</button>
+        </p>
+      </div>
+      <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28248%2C192%2C200%2C0.94%29&t=base16-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=37px&ph=42px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=13.5px&lh=130%25&si=false&es=2x&wm=false&code=%2520%253Cbutton%2520class%253D%2522btn%2520btn-primary%2520btn-round-1%2522%253EPrimary%253C%252Fbutton%253E%250A%2520%253Cbutton%2520class%253D%2522btn%2520btn-success%2520btn-round-1%2522%253ESuccess%253C%252Fbutton%253E%250A%2520%253Cbutton%2520class%253D%2522btn%2520btn-warning%2520btn-round-1%2522%253EWarning%253C%252Fbutton%253E%250A%2520%253Cbutton%2520class%253D%2522btn%2520btn-danger%2520btn-round-1%2522%253EDanger%253C%252Fbutton%253E%250A%2520%2520%250A"
+  style="width: 100%; height: 260px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+      <h3 class="avatar-heading">Link Button</h3>
+      <p class="component-navbar-item">
+        Whenever you want your user to click on a link , use link style buttons.  
+      </p>
+
+      <div class="button-items">
```

Comment:
use flex, justify around/evenly to present item on doc page so that it give good look

Reply:
okay will surely used this .Thankyou

Reply:
> use flex, justify around/evenly to present item on doc page so that it give good look

+1